README.md

```diff
@@ -1,6 +1,102 @@
 # ShopifyImportMassager
 
-**TODO: Add description**
+The script takes Shopify CSV exports and converts the order name in the csv to
```

Comment:
Because someone else will be using the code, someone who isn't familiar with Elixir, let's do some good explaining so they understand how to do things (how to run the program) and how the program works.